config/local_env.example.yml

```diff
@@ -34,3 +34,7 @@ SCHOLARSARCHIVE_ADMIN_EMAIL: 'noreply@FQDN'
 
 # The full path to the directory that BrowseEverything uses for uploads, the webserver user must have access to this directory
 BROWSEEVERYTHING_FILESYSTEM_PATH: '/tmp/browse-everything'
+
+#Recaptcha configuration
+SCHOLARSARCHIVE_RECAPTCHA_SECRET_KEY: 'asdfasdfasdfasdfasdfadsfasdfadfsfa'
```

Comment:
Don't forget to add these keys to Staging and Production when this is merged.

Reply:
👍 Will do for sure. I have it already set up along with BO's recaptcha stuff, so Im locked and loaded to get those in.